src/elements/common/flowTypes.js

```diff
@@ -25,8 +25,14 @@ type GetAvatarUrlCallback = string => Promise<?string>;
 
 type GetProfileUrlCallback = string => Promise<string>;
 
+type Page = {
+    type: 'page',
+    value: number,
+};
+
 type AdditionalVersionInfo = {
     currentVersionId?: ?string,
+    location?: Page,
```

Comment:
Hm, this doesn't really seem to fit on the version object, since it's specific to an annotation. Is this something we can derive based on the active annotation id?

Reply:
That's a good idea, let me give that a shot

Reply:
Actually that might not work because `ContentPreview` has no access to neither `feedItems` nor a list of annotations. Maybe `AdditionalVersionInfo` can contain the whole annotation object?

Reply:
The annotation isn't directly related to the selected version, though. It seems like there should be two separate callbacks, one for version change and one for annotation change.